src/utils/file-structure.js

```diff
@@ -278,8 +278,14 @@ function segregate(files) {
   };
 }
 
+function isFolder(file) { 
+  return file.type === FILE_TYPES.FOLDER
```

Comment:
You don't seem to have prettier configured, semi-column are miss-placed.

Reply:
Indeed. I can help you with this if you want @pvilalta .

Reply:
problem solved ! thanks @NicolasRichel